cpp/zotero_cgi.cc

```diff
@@ -47,8 +47,8 @@ std::string zts_url(Zotero::TranslationServer::GetUrl());
 const std::string ZTS_HARVESTER_CONF_FILE("/usr/local/ub_tools/cpp/data/zts_harvester.conf");
 const std::vector<std::pair<std::string,std::string>> OUTPUT_FORMAT_IDS_AND_EXTENSIONS {
     // custom formats
-    { "marcxml", "xml" },
-    { "marc21", "mrc" },
+    { "marcxml", "marc-xml" },
+    { "marc21", "marc-21" },
```

Comment:
this means that we would generate files with the extension ".marc-xml" and ".marc-21"???

Reply:
No, that's just internal format string used to specify the output format. zotero_cgi's output files don't have any extension (as the filenames are automatically generated) - The above format string is passed to the Zotero/MarcFormatHandler, which then instantiates the correct writer.